### PR TITLE
Added the ability to import a koinos seed phrase

### PR DESCRIPTION
**Pull request by Yonatan19981**

### CODE REVIEW

File: src/index/components/ImportSeedPhrase.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      password2: "",
+    };
+  },
+  mixins: [Storage, AlertHelper],
+  methods: {
+    async importSeedPhrase() {
+      try {
+        if (this.password1 !== this.password2)
+          throw new Error("password mismatch");
+        const ethers = require("ethers");
```
Comment (joticajulian) [inline src/index/components/ImportSeedPhrase.vue:48]: import ethers above

File: src/index/components/ImportSeedPhrase.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        const hdNode = ethers.utils.HDNode.fromMnemonic(this.mnemonic);
+        const keyNumber0 = hdNode.derivePath("m/44'/659'/0'/0/0");
+        let privateKey=keyNumber0;
+        console.log("privateKey",privateKey.privateKey)
+        const signer = Signer.fromWif(privateKey.privateKey);
```
Comment (joticajulian) [inline src/index/components/ImportSeedPhrase.vue:53]:
```
const signer = new Signer({ privateKey: keyNumber0.privateKey.slice(2) });
const privateKey = signer.getPrivateKey("wif", false);
```

File: src/index/components/ImportSeedPhrase.vue [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+<script>
+import { Signer } from "koilib";
+import router from "@/index/router";
+import Storage from "@/shared/mixins/Storage";
+import AlertHelper from "@/shared/mixins/AlertHelper";
+export default {
+  data() {
+    return {
+      privateKey: "",
```
Comment (joticajulian) [inline src/index/components/ImportSeedPhrase.vue:37]: define mnemonic here and remove privateKey

File: src/index/components/ImportSeedPhrase.vue [hunk 4]
```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="middle">
+    <div class="container">
+      <h1>Import Seed Phrase</h1>
+      <img src="" alt="" />
+      <input
+        id="seed"
+        v-model="mnemonic"
+        type="password"
```
Comment (joticajulian) [inline src/index/components/ImportSeedPhrase.vue:9]: remove password to be able to see the phrase

**Comment by Yonatan19981** [comment]
Changes updated

**Pull request merged**